spec/features/visitors/sign_up_spec.rb

```diff
@@ -82,24 +89,8 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables OTP lockout during account creation' do
```

Comment:
Yeah, you're right. I was thinking N would be a specific value as currently configured by Devise, but that wouldn't catch someone using a different value.